controllers/helmrelease_controller_test.go

```diff
@@ -278,6 +281,208 @@ invalid`,
 	}
 }
 
+func TestValidation(t *testing.T) {
```

Comment:
The tests specifically validate values references. Let's rename it to be more specific.